vividus-soft-assert/src/main/java/org/vividus/softassert/issue/KnownIssueChecker.java

```diff
@@ -142,8 +142,10 @@ boolean isPotentiallyKnown(KnownIssueIdentifier knownIssueIdentifier)
             return testInfo != null && (
                     isPotentiallyKnown(testInfo.getTestSuite(), knownIssueIdentifier.getTestSuiteCompiledPattern())
                             || isPotentiallyKnown(testInfo.getTestCase(),
-                            knownIssueIdentifier.getTestCaseCompiledPattern()) || isPotentiallyKnown(
-                            testInfo.getTestStep(), knownIssueIdentifier.getTestStepCompiledPattern()));
+                            knownIssueIdentifier.getTestCaseCompiledPattern())
+                            || (testInfo.getTestSteps() != null && !testInfo.getTestSteps().stream()
```

Comment:
when can `testInfo.getTestSteps()` be null?

Reply:
I think no, changed (in tests also)